cli/schemas/config-file.v1.json

```
@@ -300,7 +300,7 @@
               "type": "array",
               "description": "List of rule names that will be excluded from configured tag sets. If the same rule is in `include` it will be run.",
               "items": {
-                "$ref": "https://raw.githubusercontent.com/denoland/deno_lint/main/schemas/rules.v1.json"
+                "$ref": "https://deno.land/x/deno/cli/schemas/lint-rules.v1.json"
```

@nayeemrmn are these correct URLs? Which URL for the the `config-file.v1.json` file does the LSP/vscode use?

---

It uses the github URL https://github.com/denoland/vscode_deno/blob/3.42.0/package.json#L596. A relative path like this should work:
```suggestion
                "$ref": "lint-rules.v1.json"
```
If it doesn't work we can verify right after landing and then fix 🤷

---

Thanks, let's do that 👍